### PR TITLE
Rating : Component Added

### PR DESCRIPTION
**Pull request by m26unkwn**
1. Simple Rating Component Added

### CODE REVIEW

File: styles/rating.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.rating {
+  display: inline-flex;
+  gap: 6px;
+  padding: 0.5rem 1rem;
```
Comment (PuneetP16) [inline styles/rating.css]: Suggestion: you could use px units instead of rem when your value is less than 10px.

**Review by PuneetP16 — APPROVED** [review]
Rating component looks good. check the unit mentioned in the comment. 

UI suggestion: You could improve color contrast to make the rating component more pleasing to users eyes.
The spacing could also be increased by some amount.

What I liked in your code was, you are using variables for colors and code is to the point.

File: styles/rating.css (hunk 1 above)
Comment (m26unkwn) [reply styles/rating.css]: done

File: components/rating.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+              </svg>
+            </a>
+          </footer>
+        </aside>
+        <!-- Documentation of component end -->
+      </section>
+      <!-- main document ends here -->
+    </main>
+    <script type="text/javascript">
+      console.log("hello world");
```
Comment (dev-ashishsethi) [inline components/rating.html:185]: remove console.log

File: components/rating.html [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+      const closeModal_2 = document.getElementById("close-modal_2");
+      modalBtn.addEventListener("click", () => {
+        modal.classList.add("active");
+      });
+      closeModal_2.addEventListener("click", () => {
+        modal.classList.remove("active");
+      });
+      closeModal.addEventListener("click", () => {
+        modal.classList.remove("active");
+      });
```
Comment (dev-ashishsethi) [inline components/rating.html:198]: don't write inline-js

File: styles/rating.css [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.rating {
+  gap: var(--spacing-md);
+  padding: var(--spacing-sm) var(--spacing-md);
```
Comment (dev-ashishsethi) [inline styles/rating.css:3]: good that you are using variables for sizing as well

**Review by dev-ashishsethi — APPROVED** [review]
do check comments

**Pull request merged**